### DISCUSSION TITLE
Convert Draft Issues to Issues

### DISCUSSION THREAD

**Discussion by snesjhon** [community/community#4284] [Projects and Issues] [answered]
I love the idea of having Draft issue be easily create-able in the table view. However there isn't any actions associated with it, as far as I could tell (?). You can assign some fields to it, but that's about it. 

I believe in the previous Issues boards you were able to convert a Note into an Issue and I found that helpful. I wonder if this is also possible to have other actions on these Draft Issues.

**Comment by azenMatt** [comment]
We are working on this at the moment - it should be ready soon!

**Reply by snesjhon** [reply]
Wonderful. Thank you @azenMatt!

**Comment by mansona** [comment]
Hey folks, just checking if there has been any movement on this? I notice that if you create a kanban view you are able to "Convert to issue" but I don't see any corresponding action in the list view 🤔  I've only just started using the beta so it could just be that I'm missing it

**Reply by vultuk** [reply]
If you select a repository from a Draft issue it will create an issue in that repository. That's how I've been using it.

**Reply by snesjhon** [reply]
I was able to access the feature using the arrow dropdown

![image](https://user-images.githubusercontent.com/5641734/140134843-87b464fa-ae62-4ab4-aa69-86a30a97a5bc.png)

**Comment by azenMatt** [comment, marked as answer]
Hi everyone - I missed updating this discussion when the functionality went live! So for anyone else looking for how this works 

You can now convert a draft to an issue three different ways:

1 - On the table, using the arrow dropdown on the left hand side (thanks @snesjhon for the screenshot of this above)
2 - On the table, via any of the built-in-metadata cells (repo, assignee, milestone etc)
3 - On the board, using the card menu found in the `...`
